.github/workflows/rubyonrails.yml

```diff
@@ -0,0 +1,58 @@
+# This workflow uses actions that are not certified by GitHub.  They are
+# provided by a third-party and are governed by separate terms of service,
+# privacy policy, and support documentation.
+#
+# This workflow will install a prebuilt Ruby version, install dependencies, and
+# run tests and linters.
+name: "Ruby on Rails CI"
```

Comment:
Lets change the name for Tests and rubocop. Though Ideally I'd have 2 different actions. One specifically for `rubocop` and one for `rake`